lib/public/IDBConnection.php

```diff
@@ -357,11 +362,13 @@ public function migrateToSchema(Schema $toSchema): void;
 
 	/**
 	 * Returns the database provider name
+	 *
+	 * @param bool $strict differentiate between database flavors, e.g. MySQL vs MariaDB
 	 * @link https://github.com/nextcloud/server/issues/30877
 	 * @since 28.0.0
```

Comment:
Add a second since line:
```suggestion
	 * @since 32.0.0 Optional parameter $strict was added
	 * @since 28.0.0
```